app/views/homes/index.html.erb

```diff
@@ -1,4 +1,6 @@
 <p>
   Modify me in `app/views/homes/index.html.erb` or change the root directive in
   `config/routes.rb`
+
+  This is where our homepage currently goes for NOT signed in users
 </p>
```

Comment:
Either remove this page, or configure something for it. Its just a default, so you could rip it out and set a new root resource